sdks/python/apache_beam/ml/inference/sklearn_inference_regression_it_test.py

```diff
@@ -0,0 +1,67 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""End-to-End test for Sklearn Inference using the Japanese housing regression
+sample code."""
+
+import logging
+import unittest
+import uuid
+
+import pytest
+
+from apache_beam.examples.inference import sklearn_japanese_housing_regression
+from apache_beam.io.filesystems import FileSystems
+from apache_beam.testing.test_pipeline import TestPipeline
+
+
+def file_lines_sorted(filepath):
+  with FileSystems().open(filepath) as f:
+    lines = f.readlines()
+  lines = [l.decode('utf-8').strip('\n') for l in lines]
+  return sorted(lines)
+
+
+@pytest.mark.skip
+@pytest.mark.uses_sklearn
+@pytest.mark.it_postcommit
+class SklearnInferenceRegression(unittest.TestCase):
```

Comment:
@AnandInguva What is the most efficient way you've found to run these tests?  Normally I was just using the script to run locally, but since this test relies on data in gcs that is not public I was wondering if you've found an efficient way to run your tests.

Reply:
You can do one thing. 

- Remove the skip decorator and run `./gradlew :sdks:python:test-suites:direct:py38:inferencePostCommitIT`. This gradle task will cover this test since it has `uses_sklearn` marker. 

Right now it skips the all the tests since the `skip` decorator is on the class. You can move this decorator on to the test methods instead.
Also, can you add this test to the already existing `sklearn_inference_it_test.py`?.  This _it_test.py is intended for all the sklearn inference regression tests.
https://github.com/apache/beam/blob/8a32df19e2c8e6b5dd5e5142b62b6ec6db7cc2bd/sdks/python/apache_beam/ml/inference/sklearn_inference_it_test.py#L41

Reply:
OK, I made this change, it makes sense to keep these tests together. 

I wasn't able to get hte test to run with gradlew, so I just used my local files and ran the test.  All I really needed to change was a flag to modify the numbers appended to the output file and the test passed.